src/lib/template_helpers.js

```diff
@@ -0,0 +1,23 @@
+export default {
+  stripHtml: () => {
+    // Strip all HTML from the passed string.
+    return (text, render) => {
+      let tmp = document.createElement('DIV');
+      tmp.innerHTML = text;
```

Comment:
@chuckharmston To address the specific bug here: `render(text)` is the value that you want.

Taking a step back, do we really need to do this at all? Hitting the DOM to filter text is a real performance hit, while the underlying issue is that we have some (accurately placed) bold tags coming back from the server. How about we just leave those `<b>` tags in there, for now, and plan in the future to have services return plain text, not html?